src/protocols/secure_channel/PASESession.cpp

```diff
@@ -979,4 +981,31 @@ CHIP_ERROR PASESession::OnMessageReceived(ExchangeContext * exchange, const Payl
     return err;
 }
 
+CHIP_ERROR PASEVerifier::Serialize(PASEVerifierSerialized & output)
+{
+    memcpy(output, mW0, kSpake2p_WS_Length);
+    memcpy(&output[kSpake2p_WS_Length], mL, kSpake2p_WS_Length);
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR PASEVerifier::Deserialize(const PASEVerifierSerialized & input)
+{
+    memcpy(mW0, input, kSpake2p_WS_Length);
+    memcpy(mL, &input[kSpake2p_WS_Length], kSpake2p_WS_Length);
+
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR PASEVerifier::Deserialize(const ByteSpan input)
+{
+    VerifyOrReturnError(!input.empty(), CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
If the size check below passes, this condition must pass, too. It can be removed.